_posts/2019-09-16-picture-archivist-in-f.html

```diff
@@ -306,7 +306,7 @@ <h3 id="abe95ba6865745bc9df8004079d8a250">
 		Calculating moves <a href="#abe95ba6865745bc9df8004079d8a250" title="permalink">#</a>
 	</h3>
 	<p>
-		One pure step remains. The result of calling the <code>moveTo</code> function is a tree with the desired structure. In order to actually move the files, though, for each file you'll need to keep track of both the source path and the destination path. To make that explicit, you can define a type for that purpose:
+		One pure step remains. The result of calling the <code>moveTo</code> function given a tree with the desired structure. In order to actually move the files, though, for each file you'll need to keep track of both the source path and the destination path. To make that explicit, you can define a type for that purpose:
```

Comment:
This isn't a typo 😄

Reply:
Oh, sure.  I see it now.  Thanks for pointing this out.  I removed the commit from this branch that made this change.  There is still the other commit (that I think fixes a typo ;) ).